src/map/map.ts

```diff
@@ -0,0 +1,38 @@
+import Dispatcher from '@/stores/Dispatcher'
+import { Map, View } from 'ol'
+import { fromLonLat } from 'ol/proj'
+import { MapIsLoaded } from '@/actions/Actions'
+import { defaults as defaultControls } from 'ol/control'
+
+let map: Map | undefined
+
+export function createMap(): Map {
+    map = new Map({
+        view: new View({
+            multiWorld: false,
```

Comment:
We still need to restrict map zoom/pan so we cannot pan around the globe?! (currently we can)

Reply:
I don't have an oppinion on that

Reply:
When we pan around the globe: will the longitudes get an offset of +360°? For leaflet this was the case and our API has issues with that so we limited the paning for leaflet.

Reply:
(will try this when I try out this PR)

Reply:
No, I panned around the world a few times and the longitude of the requests seems to always be in the range -180, 180

Reply:
Yes, perfect :+1:

Reply:
We can limit the panning like this: 

```typescript
    const sw = fromLonLat([-180, -75])
    const ne = fromLonLat([180, 85])
    map = new Map({
        view: new View({
            extent: [sw[0], sw[1], ne[0], ne[1]],
```

but I don't really have a preference here